### PR TITLE
Write user guide for Update Workout

### PR DESCRIPTION
**Pull request by TianaiYan**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -118,6 +118,54 @@ Showed all workouts in list
...
 
+### Update a Workout: `workout /update`
+Modifies an existing workout from the workout list.<br>
+*Noted: Only the number of repetition can be changed.*
+
+Format: `workout /update <index of workout to update> <new number of repetitions>`
+
+| Parameters                     | Description                                              |
+|--------------------------------|----------------------------------------------------------|
+| `<index of workout to update>` | The index of workout to be update from the workout list. |
```
Comment (Musfirahe0556596) [inline docs/UserGuide.md:129]: Minor language issue - "The index of workout to be updated..."

**Review by Musfirahe0556596 — APPROVED** [review]
UG looks great! Thank you :)

File: docs/UserGuide.md [hunk 2]
```diff
@@ -118,6 +118,54 @@ Showed all workouts in list
...
+```
+----------------------------------------------------------------------
+Showing workouts 1-3 of 3:
+1. push up (10 reps)
+2. sit up (10 reps)
+3. pull up (10 reps)
+Showed all workouts in list
+----------------------------------------------------------------------
+```
+To update workout with index 1 to 15 reps, the following command should be entered:
```
Comment (alanlowzies) [inline docs/UserGuide.md:143]: Perhaps the second part of the sentence could be rephrased to '..., enter the following command:'? The original phrase sounds like a suggestion, which sounds a bit off in this context.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -118,6 +118,54 @@ Showed all workouts in list
...
+**Expected Outcome**
+```
+----------------------------------------------------------------------
+Alright, the following workout has been updated:
+
+	push up (15 reps)
+
+----------------------------------------------------------------------
+```
+Now workout with index 1 (push up with 10 reps) is updated to push up with 15 reps.
```
Comment (alanlowzies) [inline docs/UserGuide.md:156]: Consider adding the word 'the' in-between 'Now' and 'workout'.

**Review by alanlowzies — COMMENTED** [review]
General content is good. Just some grammar-related suggestions for you to consider. Thank you

File: docs/UserGuide.md (hunk 1 above)
Comment (TianaiYan) [reply docs/UserGuide.md:129]: Didn't notice that. Thank you for pointing it out.

File: docs/UserGuide.md (hunk 2 above)
Comment (TianaiYan) [reply docs/UserGuide.md:143]: Thank you for your suggestions.

**Pull request merged**